packages/core/defaultTheme.js

```diff
@@ -69,10 +69,15 @@ const buttonsStyles = {
   DEFAULT: "default"
 };
 
+const entities = {
```

Comment:
This may end up becoming a temporary `const`. I imagine we might end up moving in a direction which utilized an <Icon> component in the future. For now, this works.